### DISCUSSION TITLE
Is it possible to delete an account without deleting the repositories?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#81018] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Due to the new feed and 2FA obligation I want to delete my account, but I don't want to delete the repositories as someone might be using them. 

Can I unsubscribe and keep the repositories?

**Comment by Nupurluthra** [comment]
Yes, it is possible to delete a user account on platforms like GitHub without deleting the repositories. The repositories will become orphaned but won't be automatically deleted.

**Reply by ghost** [reply]
Today I went to delete my account and I get this message:

"We will immediately delete all of your repositories (6), along with all of your forks, wikis, issues, pull requests, and GitHub Pages sites.

You will no longer be billed, and after 90 days your username will be available to anyone on GitHub."
